File: ember-projects/linl/tmp/remover-tmp_dest_dir-7G5MFXbM.tmp/models/residence.js
import DS from 'ember-data';

export default DS.Model.extend({
  user: DS.belongsTo('user'),
  name: DS.attr('string'),
  kind: DS.attr('string'),
  monthly_payment_cents: DS.attr('number'),
  value_cents: DS.attr('number'),
  mortgage_balance_cents: DS.attr('number'),
  years_remaining: DS.attr('number'),
  misc_annual_expenses: DS.attr('number'),
  utilities_cents: DS.attr('number'),
  created_at: DS.attr('date'),
  updated_at: DS.attr('date')
});

File: rails-projects/linl-api/app/admin/residence.rb
ActiveAdmin.register Residence do
  permit_params :id, :user_id, :name, :kind, :monthly_payment_cents, :value_cents, 
    :mortgage_balance_cents, :years_remaining, :misc_annual_expenses_cents,
    :utilities_cents, :created_at, :updated_at
end
